Tests/conf.json

```diff
@@ -49,10 +49,7 @@
           "playbookID": "devo_test_playbook"
         },
         {
-            "integrations": [
-                "VirusTotal",
-                "urlscan.io"
-            ],
+          "integrations": "VirusTotal",
```

Comment:
Why are you using `VirusTotal` and no `urlscan` ?

Reply:
Switched to urlscan.